microsoft-edge/devtools-guide-chromium/memory-problems/index.md

```diff
@@ -250,7 +250,27 @@ DevTools shows you a breakdown of memory allocation by function.  The default vi
 
 ![Allocation sampling.](../media/memory-problems-glitch-example-05-memory-allocation-sampling-heavy-bottom-up.msft.png)
 
+## Use settings for Allocation sampling to reduce garbage
 
+By default, the **Allocation sampling** profiling type only reports allocations that are still alive at the end of the recording session.  Objects that are generated, garbage collected (GC'd), and then disappear aren't tracked by either **Allocation sampling** or **Allocation instrumentation on timeline**.
+
+You can configure the **Allocation sampling** profiling type to track objects that are discarded by GC with settings.
+
+1. Click the **Allocation sampling** radio button.
+
+1. Click the **Include objects discarded by major GC** and **Include objects discarded by minor GC** settings.
+
+    ![Allocation sampling GC settings.](../media/memory-problems-memory-allocation-sampling-gc-settings.png)
```

Comment:
Make window as narrow as feasible while still looking recognizable/normal.  That linearly increases crispness, up to a point; eventually the resulting png is rendered narrower than the content area, so don't need to make window extremely narrow before screenshotting it.

Here is GitHub's view - different sizing than LMC, but even in this wide view, window needs to be narrower before screenshotting it.
https://github.com/MicrosoftDocs/edge-developer/blob/user/zoghadya/allocation-sampling-gc-settings/microsoft-edge/devtools-guide-chromium/memory-problems/index.md#use-settings-for-allocation-sampling-to-reduce-garbage